modules/test/playwright/tests/journal-web/pages/JournalPage.ts

```diff
@@ -106,6 +114,29 @@ export class JournalPage {
 		});
 	}
 
+	async goToJournalFolderAction(action: string, title: string) {
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {name: 'cards'}),
+			trigger: this.page.getByLabel('Select View, Currently Selected: '),
+		});
+
+		const folder = this.page.locator(`dd[data-title="${title}"]`);
```

Comment:
In this case is quite the opposite, this selector feel too much general.
Only a `dd` element `data-title` feel fragile, let me check if we have more semantic `data`, `class` or `attr`.

Reply:
Maybe something like this would work better:

```js
const folder = this.page.locator(`[data-qa-id="row"][data-title="${title}"]`);
```

![image](https://github.com/user-attachments/assets/469410aa-00d5-4377-b37d-73a9386e47f3)

Reply:
working with const folder = this.page.locator(`[data-qa-id="row"][data-title="${title}"]`);